packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -100,13 +102,46 @@ async function withPersistence(
     clientId,
     platform,
     queue,
-    serializer
+    serializer,
+    settings.experimentalTabSynchronization
   );
 
   await fn(persistence, platform, queue);
   await persistence.shutdown();
 }
 
+async function withPersistence(
+  clientId: ClientId,
+  fn: (
+    persistence: IndexedDbPersistence,
+    platform: TestPlatform,
+    queue: AsyncQueue
+  ) => Promise<void>
+): Promise<void> {
+  return withCustomPersistence(
+    clientId,
+    new PersistenceSettings(/* enabled */ true),
+    fn
+  );
+}
+
+async function withMultiTabPersistence(
```

Comment:
One last suggestion if you like to add more iteration to this PR: The code refers to this feature as "multiclient" rather than "multitab", since it could technically be ported to the other platforms. Maybe. Probably not.